utils.py

```diff
@@ -1113,23 +1113,26 @@ def user_is_owner(asset_data: Optional[dict] = None) -> bool:
     return False
 
 
-def asset_from_newer_blender_version(asset_data):
+def asset_from_newer_blender_version(asset_data, blender_version=None):
```

Comment:
If this func is meant to be universal, it should be implemented universally.

What if blender_version is not a tupple containing 3 items, but less?